frontend/ui/lib/widgets/create_trip_widget.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+
+class CreateTripWidget extends StatefulWidget {
+  CreateTripWidget({Key key}) : super(key: key);
+  @override
+  CreateTripState createState() => CreateTripState();
+}
+
+class CreateTripState extends State<CreateTripWidget> {
```

Comment:
Nit: restrict visibility by prefixing this internal class name with an underscore: `_CreateTripState`. (Note that this isn't a huge deal because we don't currently break the UI into different packages/libraries, but when/if we do, this becomes important for abstraction isolation.)